testing/hosts/default/etc/sysctl.conf

```diff
@@ -1,6 +1,6 @@
 #
 # /etc/sysctl.conf - Configuration file for setting system variables
-# See /etc/sysctl.d/ for additonal system variables
```

Comment:
This is a modified distribution file, as it was provided by Debian wheezy. The fix for the typo was included in Debian jessie, on which the current testing environment is based, so I guess we can apply that now too (I explicitly excluded the file from codespell runs so far).